### PR TITLE
combine prepare_env scripts into one

### LINKED ISSUE #33: combine prepare_env.sh and prepare_env_wsl.sh into one script
combine prepare_env.sh and prepare_env_wsl.sh into one script would help user  easily to prepare the environment of GraphScope.

### PR DESCRIPTION
**Pull request by acezen**
## What do these changes do?
combine the prepare_env.sh and prepare_env_wsl.sh into one

## Related issue number

Fixes #33

### CODE REVIEW

**Comment by acezen** [comment]
cc/ @yecol @sighingnow please review and give some comments.

**Review by sighingnow — APPROVED** [review]
LGTM, just a little concern about `WSL_DISTRO_NAME` or `IS_WSL`? Are they guaranteed to exist under WSL2 environment?

File: scripts/prepare_env.sh [hunk 1]
```diff
@@ -43,8 +43,45 @@ function install_dependencies() {
   curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/"${K8S_VERSION}"/bin/linux/amd64/kubectl && \
   chmod +x kubectl && sudo mv kubectl /usr/local/bin/ && sudo ln /usr/local/bin/kubectl /usr/bin/kubectl || true
 
-  curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 && \
-  chmod +x minikube && sudo mv minikube /usr/local/bin/ && sudo ln /usr/local/bin/minikube /usr/bin/minikube || true
+  if [[ ! -z "$IS_WSL" && ! -z "$WSL_DISTRO_NAME" ]]; then
```
Comment (siyuan0322) [inline scripts/prepare_env.sh:46]: Nits: Consider extract the the common part `! -z "$IS_WSL" && ! -z "$WSL_DISTRO_NAME"` into one boolean variable so we don't need to evaluate it multiple times, and will be more clear.
And please add curly braces around `IS_WSL` and `WSL_DISTRO_NAME` like the other variables do.

**Comment by acezen** [comment]
> LGTM, just a little concern about `WSL_DISTRO_NAME` or `IS_WSL`? Are they guaranteed to exist under WSL2 environment?

there is a issue in WSL that suggest to use the env to check is in WSL or not. https://github.com/microsoft/WSL/issues/4555#issuecomment-700213318

File: scripts/prepare_env.sh [hunk 2]
```diff
@@ -43,8 +43,60 @@ function install_dependencies() {
...
+    if [[ ! -z ${IS_WSL} && ! -z ${WSL_DISTRO_NAME} ]]; then
+      sudo systemctl start docker
+    else
+      sudo dockerd > /dev/null& || true
+    fi
+  fi
+}
+
+function launch_k8s_cluster() {
+  if [[ ! -z ${IS_WSL} && ! -z ${WSL_DISTRO_NAME} ]]; then
```
Comment (sighingnow) [inline scripts/prepare_env.sh:67]: I think you need `"${IS_WSL}"` rather than `{IS_WSL}`. As well for `WSL_DISTRO_NAME`.

Otherwise LGTM.

**Review by sighingnow — APPROVED** [review]

**Review by sighingnow — CHANGES_REQUESTED** [review]
Please update README.md which refers `prepare_env_wsl.sh` (and maybe tutorial, references, etc.)

image

File: scripts/prepare_env.sh [hunk 3]
```diff
@@ -43,8 +44,60 @@ function install_dependencies() {
...
+    # mount $HOME dir to cluster container, which is kind-control-plane
+    sed -i 's@/path/to/my/files/@'"${HOME}"'@g; s@/files@'"${HOME}"'@g' ./config-with-mounts.yaml  || true
+    sudo kind create cluster --config config-with-mounts.yaml
+    sudo cp -r /root/.kube ${HOME} || true
+    sudo chown -R "$(id -u)":"$(id -g)" "${HOME}"/.kube || true
+  fi
+}
+
+function pull_images() {
+  sudo docker pull registry.cn-hongkong.aliyuncs.com/graphscope/graphscope:latest || true
```
Comment (yecol) [inline scripts/prepare_env.sh:89]: echo "pulling graphscope images...."

**Review by yecol — CHANGES_REQUESTED** [review]
I suggest the script could be more verbose.
- tell user what is doing.
- give a summary at the end of the entire process. 

e.g.,
```
minikube VERSION installed.
# or 
kind VERSION installed.
dockerd started.
...
```

**Comment by acezen** [comment]
> I suggest the script could be more verbose.
> 
> * tell user what is doing.
> * give a summary at the end of the entire process.
> 
> e.g.,
> 
> ```
> minikube VERSION installed.
> # or 
> kind VERSION installed.
> dockerd started.
> ...
> ```
 already add some verbose log and a summary to script, please review again

**Review by sighingnow — APPROVED** [review]

**Review by yecol — APPROVED** [review]

**Pull request merged**